tools/NuGet/template-nuget/DynamoVisualProgramming.ZeroTouchLibrary.nuspec

```diff
@@ -16,6 +16,9 @@
             <group targetFramework="$TargetFramework$">
                 <dependency id="DynamoVisualProgramming.DynamoServices" version="$Version$"/>
             </group>
```

Comment:
Do we need to remove this block?

Reply:
I think we need both target framework groups : 
netstandard2.0 and whatever current dynamo will target
netstandard2.0 will not have the DynamoUnits assembly (since that is targeting net6)